aries-core/src/js/components/helpers/Anchors/Anchor.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Anchor as GrommetAnchor, Box, Text } from 'grommet';
+
+export const Anchor = ({ label, size, weight, ...rest }) => {
+  return (
+    // Bottom border allows for anchor underline without having
+    // the underline intercept with descenders (portion of text
+    // that extends below baseline)
+    <Box border={{ color: 'text', side: 'bottom' }} alignSelf="start">
+      <GrommetAnchor
+        label={
+          <Text weight={weight} size={size}>
+            {label}
+          </Text>
+        }
+        {...rest}
+      />
+    </Box>
+  );
+};
+
+Anchor.propTypes = {
+  label: PropTypes.string,
```

Comment:
In grommet, in addition to a string we can pass `Anchor` a `label` of a JSX element, should we make that consistent here?

Reply:
The reason I had only put string is because I have embedded the label within the `<Text />` so that we could control the weight on the text which is something that can't be done directly on anchor. In that case, the label you are passing to it should just be a string because it will be nested inside of Text. However, I can pull the `<Text />` block out and apply it directly on my use of Anchor in `SubmitFeedback`. If we plan to make this Anchor more general, then that would be the right move. I think my approach here aligned with my thought that this was somewhat opinionated for the use case of this kind of anchor with underline.

Reply:
Sounds good. Makes sense to me.